packages/backend-ai/src/get_message.test.ts

```diff
@@ -0,0 +1,255 @@
+import { getMessage } from './get_message.js';
+import { ContentBlock, GetMessageInput } from './types.js';
+import { before, describe, it, mock } from 'node:test';
+import assert from 'node:assert';
+import {
+  BedrockRuntimeClient,
+  ConverseCommandOutput,
+} from '@aws-sdk/client-bedrock-runtime';
+
+void describe('getMessage', async () => {
+  let mockBedrockRuntimeClient: BedrockRuntimeClient;
+
+  before(() => {
+    const mockResponse: ConverseCommandOutput = {
+      output: {
+        message: {
+          role: 'assistant',
+          content: [{ text: 'This is a mock response from Bedrock.' }],
+        },
+      },
+      stopReason: 'stop_sequence',
+      usage: {
+        inputTokens: 10,
+        outputTokens: 20,
+        totalTokens: 30,
+      },
+      metrics: {
+        latencyMs: 100,
+      },
+      $metadata: {},
+    };
+
+    mockBedrockRuntimeClient = {
+      send: mock.fn(async () => mockResponse),
+    } as unknown as BedrockRuntimeClient;
+
+    mock.method(
+      BedrockRuntimeClient.prototype,
+      'send',
+      mockBedrockRuntimeClient.send
+    );
```

Comment:
I think this mock will become cleaner if we follow a pattern for the handler similar to the pattern [here](https://github.com/aws-amplify/amplify-backend/blob/f5eeb67d840a194ffeeb585bfa0ba9468c1f6cda/packages/backend/src/engine/branch-linker/lambda/branch_linker.ts#L140-L150). This will also be more performant because we aren't constructing a new client on each lambda request